src/uwtools/config.py

```diff
@@ -752,3 +753,22 @@ def create_config_obj(
                     raise ValueError(err_msg)
                 # Dump to file:
                 dump_method(path=outfile, cfg=config_obj)
+
+
+def print_config_section(config: dict, section_path: List[str], log: Logger) -> None:
+    """
+    Descends into the config via the given section keys, then prints the contents of the located
+    subtree as key=value pairs, one per line.
+    """
+    for key in section_path:
+        if isinstance(config[key], dict):
+            config = config[key]
+        else:
+            log.error(f"{key} type must be a dictionary")
+            raise UWConfigError("Key type must be a dictionary")
```

Comment:
`Key` -> `Value`, since we're testing the type of `config[key]`, which is the value?